packages/fxa-settings/src/pages/Signin/utils.ts

```diff
@@ -339,19 +339,14 @@ const getUnverifiedNavigationTarget = (
 ) => {
   const { verificationReason, verificationMethod } =
     navigationOptions.signinData;
-  const { integration, queryParams } = navigationOptions;
-  const isOAuth = isOAuthIntegration(integration);
+  const { queryParams } = navigationOptions;
 
   const getUnverifiedNavTo = () => {
-    // TODO in FXA-9177 Consider storing state in Apollo cache instead of location state
-    if (
-      ((verificationReason === VerificationReasons.SIGN_IN ||
-        verificationReason === VerificationReasons.CHANGE_PASSWORD) &&
-        verificationMethod === VerificationMethods.TOTP_2FA) ||
```

Comment:
Not sure I fully understand why we can get rid of all of these, wouldn't `CHANGE_PASSWORD` verification go through the Signin page etc.?

Reply:
Never mind, I think we've handled all these cases elsewhere now. 👍